### PR TITLE
Enhance an feature of `if-then-else` to make expressions

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Added a new parameter which is `lhs` to enhance `if-then-else` to make expressions.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ExtFilter/Scripts/ExtFilter/README.md [hunk 1]
```diff
@@ -8049,6 +8049,7 @@ If `if` condition is not given or returns any value, `then` operation is execute
 | if | expressions | (Optional) `if` condition |
 | then | expressions | Conditions to execute if `if` condition is not given or returns any value. |
 | else | expressions | (Optional) Conditions to execute if `if` returns `null`. |
+| lhs | expressions | (Optional) The value used on the left hand side instead of using the current element. |
```
Comment (ShacharKidor) [inline Packs/ExtFilter/Scripts/ExtFilter/README.md:8052]: Not sure I understood what the 'lhs' does ... Can you provide a more detailed or clearer explanation?

 In addition, you can change line 8048 with the following:
 | --- | --- | --- |
 that will create an MD table (you can change that in many places in the readme file if you like).

File: Packs/ExtFilter/ReleaseNotes/1_1_11.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+- **ExtFilter**
+  - Enhance an feature of `if-then-else` to make expressions
```
Comment (ShacharKidor) [inline Packs/ExtFilter/ReleaseNotes/1_1_11.md:4]: Maybe write about the 'lhs' and the functionality that was added

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @spearmin10, 
nice job!

I added a few suggestions. 
In addition, I'll be happy to have a short demo to better understand the new parameter that was added. 

I'm available for every question :)

File: Packs/ExtFilter/ReleaseNotes/1_1_11.md (hunk 2 above)
Comment (ShacharKidor) [reply Packs/ExtFilter/ReleaseNotes/1_1_11.md:4]: - Added the **lhs** parameter to the  **if-then-else** in order to make expressions  (explain here)

**Comment by ShacharKidor** [comment]
Hi @spearmin10, 
As discussed in the Demo, please correct the ReleaseNote and the README files according to my suggestions, and let me know when you are done. 

I'm available for any question :)

**Comment by spearmin10** [comment]
@ShacharKidor OK, I've updated them.

**Review by ShacharKidor — APPROVED** [review]
Well done!

**Pull request merged**